<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ConfigureParallelDialog</class>
 <widget class="QDialog" name="ConfigureParallelDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>500</width>
    <height>204</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Configure parallel port</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QGroupBox" name="frmParallel">
     <property name="title">
      <string>Parallel port</string>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="0" column="0" colspan="2">
       <widget class="QLabel" name="lblFileName">
        <property name="text">
         <string>File name:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0" colspan="2">
       <widget class="QLineEdit" name="txtFileName"/>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="lblFileDirectory">
        <property name="text">
         <string>File directory:</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLineEdit" name="txtFileDirectory">
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QToolButton" name="btnChooseDirectory">
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="../../res/resource.qrc">
          <normaloff>:/images/file-open_container_folder.png</normaloff>:/images/file-open_container_folder.png</iconset>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>175</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="btnOk">
       <property name="text">
        <string>Ok</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnCancel">
       <property name="text">
        <string>Cancel</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>frmParallel</tabstop>
  <tabstop>txtFileName</tabstop>
  <tabstop>txtFileDirectory</tabstop>
  <tabstop>btnChooseDirectory</tabstop>
  <tabstop>btnOk</tabstop>
  <tabstop>btnCancel</tabstop>
 </tabstops>
 <resources>
  <include location="../../res/resource.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>btnOk</sender>
   <signal>clicked()</signal>
   <receiver>ConfigureParallelDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>462</x>
     <y>165</y>
    </hint>
    <hint type="destinationlabel">
     <x>316</x>
     <y>101</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btnCancel</sender>
   <signal>clicked()</signal>
   <receiver>ConfigureParallelDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>562</x>
     <y>165</y>
    </hint>
    <hint type="destinationlabel">
     <x>316</x>
     <y>101</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btnChooseDirectory</sender>
   <signal>clicked()</signal>
   <receiver>ConfigureParallelDialog</receiver>
   <slot>slotChooseDirectory()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>596</x>
     <y>130</y>
    </hint>
    <hint type="destinationlabel">
     <x>316</x>
     <y>101</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>slotChooseDirectory()</slot>
 </slots>
</ui>
